File: Programmers/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
       String result = "";
        int turn = 0;
        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) == ' ') {
                turn = 0;
                result += " ";
            }
            else if (turn == 0) {
                if (s.charAt(i) >= 97) {
                    char tmp = (char) (s.charAt(i)-32);
                    result += tmp;
                }
                else result += s.charAt(i);
                turn = 1;
            }
            else if (turn == 1) {
                if (s.charAt(i) <= 90) {
                    char tmp = (char) (s.charAt(i)+32);
                    result += tmp;
                }
                else result+= s.charAt(i);
                turn = 0;
            }
        }
        return result;
    }
}

File: Programmers/이상한 문자 만들기.py
def solution(s):
    arr = list(s.split(" "))
    for i in range(len(arr)):
        new_str = ""
        for j in range(len(arr[i])):
            if j%2 != 0:
                new_str += arr[i][j].lower()
            else:
                new_str += arr[i][j].upper()
        arr[i] = new_str
    result = ""
    for i in range(len(arr)):
        result += arr[i]
        if i < len(arr)-1:
            result += " "
    return result
